queries.sql

```diff
@@ -53,4 +53,29 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) 
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+-- Alter to add new column
+ALTER TABLE animals
+ADD COLUMN species varchar(255);
+
```

Comment:
- [ ] according to the requirements Everything that creates or change the tables structure should be saved in the schema.sql file.